send-transaction-service/src/send_transaction_service.rs

```diff
@@ -459,7 +459,10 @@ impl SendTransactionService {
                     Self::send_transactions_in_batch(
                         &tpu_address,
                         &mut transactions,
-                        leader_info_provider.lock().unwrap().get_leader_info(),
+                        leader_info_provider
+                            .lock()
+                            .unwrap()
+                            .get_leader_info(connection_cache.use_quic()),
```

Comment:
```suggestion
                            .get_leader_info(connection_cache.protocol()),
```

to go along with behzad's suggestion

Reply:
Done